.NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/JapaneseHolidayParserConfiguration.cs

```diff
@@ -146,6 +147,8 @@ public string SanitizeYearToken(string yearStr)
 
         private static DateObject NewYear(int year) => new DateObject(year, 1, 1);
 
+        private static DateObject ChineseNewYear(int year) => new DateObject(year, 1, 1);
```

Comment:
This is not correct. Chinese New Year is the Spring Festival.

Reply:
yes but in Chinese Spring Festival has the same resolution.

Reply:
We need to fix this. Please open an issue. Don't fix it as part of this PR.